testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveTransactionalTable.java

```diff
@@ -1432,6 +1433,71 @@ public void testAcidUpdateWithSubqueryAssignment()
         });
     }
 
+    @Test(groups = HIVE_TRANSACTIONAL, timeOut = TEST_TIMEOUT)
+    public void testAcidUpdateDuplicateUpdateValue()
+    {
+        withTemporaryTable("test_update_bug", true, false, NONE, tableName -> {
+            onTrino().executeQuery(
+                    format("CREATE TABLE %s (", tableName) +
+                            " yyyy integer," +
+                            " week_number integer," +
+                            " week_start_date date," +
+                            " week_end_date date," +
+                            " genre_summary_status smallint," +
+                            " shop_summary_status smallint)" +
+                            " WITH (transactional = true)");
+
+            log.info("Inserting into table");
```

Comment:
These log messages are not particularly useful and will just clutter the test output. Please remove them.

Reply:
@findepi made the same comment.  I've pushed changes that remove them from this method.

Reply:
I removed them and force-pushed.